redbot/cogs/mod/settings.py

```diff
@@ -48,6 +50,7 @@ class ModSettings(MixinMeta):
             msg += _("Reinvite on unban: {yes_or_no}\n").format(
                 yes_or_no=_("Yes") if reinvite_on_unban else _("No")
             )
+            msg += _("Default days: {num_days} days\n").format(num_days=default_days)
```

Comment:
This isn't clear enough, after discussion in discord we decided to suggest just using Discord's terminology here, but if you have better idea on how to word this, feel free to use it.
```suggestion
            if default_days:
                msg += _(
                    "Default message history delete on ban: Previous {num_days} days\n"
                ).format(num_days=default_days)
            else:
                msg += _("Default message history delete on ban: Don't delete any\n")
```